### PR TITLE
Allow networked imports

### PR DESCRIPTION
**Pull request by theengineear**
Hello, I was looking to try out this repo in a project, but I'm only using native ES6 imports, which prevented me from trying this out. I took a look and it seemed like it would be pretty simple to open a PR to allow this sort of usage.

This PR:

1. removes `lodash` (seemed pretty easy to support things natively)
2. adds support for usage via native ES6 imports

No rush on this, but I was going to make these updates anyhow and figured you might be interested in them as well 👍

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,12 +53,12 @@ Currently just four SQL dialects are supported:
...
 sqlFormatter.format("SELECT * FROM tbl WHERE foo = @foo", {
     params: {foo: "'bar'"}
-}));
+});
 
 // Indexed placeholders
 sqlFormatter.format("SELECT * FROM tbl WHERE foo = ?", {
     params: ["'bar'"]
-}));
+});
```
Comment (theengineear) [inline README.md]: just removing extra `)`

File: README.md [hunk 2]
```diff
@@ -77,6 +77,14 @@ WHERE
 If you don't use a module bundler, clone the repository, run `npm install` and grab a file from `/dist` directory to use inside a `<script>` tag.
 This makes SQL Formatter available as a global variable `window.sqlFormatter`.
 
+To use native imports, clone and then import files directly within ES6 modules:
+
+```js
+import sqlFormatter from '<PATH-TO-CLONED-REPO>/src/sqlFormatter.js';
```
Comment (theengineear) [inline README.md]: This is the use case I'm looking to support.

File: src/core/Indentation.js [hunk 3]
```diff
@@ -26,7 +23,7 @@ export default class Indentation {
      * @return {String}
      */
     getIndent() {
-        return repeat(this.indent, this.indentTypes.length);
+        return this.indent.repeat(this.indentTypes.length);
```
Comment (theengineear) [inline src/core/Indentation.js]: `.repeat` is supported by modern browsers: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/repeat

File: src/core/Tokenizer.js [hunk 4]
```diff
@@ -1,6 +1,9 @@
-import isEmpty from "lodash/isEmpty";
-import escapeRegExp from "lodash/escapeRegExp";
-import tokenTypes from "./tokenTypes";
+import tokenTypes from "./tokenTypes.js";
+
+// See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions
```
Comment (theengineear) [inline src/core/Tokenizer.js]: For comparison: https://github.com/lodash/lodash/blob/master/escapeRegExp.js

File: package-lock.json [hunk 5]
```diff
@@ -4163,7 +4163,8 @@
     "lodash": {
       "version": "4.17.11",
       "resolved": "https://registry.npmjs.org/lodash/-/lodash-4.17.11.tgz",
-      "integrity": "sha512-cQKh8igo5QUhZ7lg38DYWAxMvjSAKG0A8wGSVimP07SIUEK2UO+arSRKbRZWtelMtN5V0Hkwh5ryOto/SshYIg=="
+      "integrity": "sha512-cQKh8igo5QUhZ7lg38DYWAxMvjSAKG0A8wGSVimP07SIUEK2UO+arSRKbRZWtelMtN5V0Hkwh5ryOto/SshYIg==",
+      "dev": true
```
Comment (theengineear) [inline package-lock.json]: importantly, `"dev": true`

**Comment by nene** [comment]
The part of removing lodash has been integrated and released as part of 3.1.0.

However, I'm for now rejecting the rest of this PR.

**Comment by theengineear** [comment]
That's fine — thanks for taking a look anyhow!

**Pull request closed without merging**